packages/klevu-ui/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "dependencies": {
     "@floating-ui/dom": "^1.4.5",
-    "@klevu/core": "3.4.0",
+    "@klevu/core": "file:../klevu-core",
```

Comment:
is this ok to have here?

Reply:
Will change when I know what version we will have.